lib/Takeaway.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'twilio'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)